cli/tests/pcluster/models/test_imagebuilder_extra_attributes.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+
+import pytest
+from assertpy import assert_that
+
+from pcluster.models.imagebuilder_extra_attributes import ChefAttributes
+from tests.pcluster.models.imagebuilder_dummy_model import imagebuilder_factory
+
+
+@pytest.mark.parametrize(
+    "resource, dna_json",
+    [
+        (
+            {
+                "dev_settings": {
+                    "cookbook": {
+                        "chef_cookbook": "file:///test/aws-parallelcluster-cookbook-3.0.tgz",
+                        "extra_chef_attributes": '{"nvidia": { "enable" : "true" }, "dcv" :"false"}',
```

Comment:
I know that it's just a test and doesn't matter but lets change "enable" to "enabled"

Reply:
Done